srcopsmetrics/kebechet_metrics.py

```diff
@@ -141,15 +142,21 @@ def _get_update_manager_pull_requests(self):
 
             data.append([created_at, pr_type, ttm, ttfr, tta, merged_by_kebechet_bot, rejected_by_kebechet_bot])
 
+        if not data:
+            return []
```

Comment:
this one should be an empty dataframe, so `pd.DataFrame()` to be consistent.

Reply:
yes, changed to empty dataframe, other related checks to that also added